src/main.rs

```diff
@@ -1,3 +1,22 @@
-fn main() {
-    println!("Hello, Gorillas!");
+extern crate ggez;
+use ggez::*;
+
+mod intro;
+
+
+pub fn main() {
+    let mut cfg = conf::Conf::new(); // TODO: Load from configuration file
+    cfg.window_setup.title = "Rusty Gorillas".to_string();
+    cfg.window_mode.width = 640.0;
+    cfg.window_mode.height = 480.0;
+
+    let context_builder = ggez::ContextBuilder::new("super_simple", "ggez").conf(cfg);
+    let (ctx, event_loop) = &mut context_builder.build().unwrap();
+    
+    let state = &mut intro::State::new(ctx).unwrap();
+
+    match event::run(ctx, event_loop, state) {
+        Ok(_) => (),
+        Err(e) => println!("Error occured: {}", e)
```

Comment:
I'd also suggest to exit the 1 return code to indicate error.

Reply:
Very good point again, addressed in 7e6f8eecbaeef223f843af0373c4e0e80a3882ee